### PR TITLE
updates to jquery, de-railsing, structure

### PR DESCRIPTION
**Pull request by scottcreynolds**
remove ruby/rails stuff, update examples, handle #14 and #13 and #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,168 +1,187 @@
...
 
-## Intro
-
-So far, you've been using a gem to access a web API.  Gems allow us to easily access/modify the data that's important to us. But what happens if the API we want to use doesn't have an associated gem?  We will need to handle everything ourselves. If that's the case, we'd better learn more about what an API really is!
-
-## API Basics
+## Introduction
 
-We interact with web APIs through a set of urls. Each url defines a resource that we access.  The benefit of using a gem is that we don't have to build these urls ourselves.  The gems wrap the necessary code into neat little packages.  When we ask a gem for data, a gem will make a request to the API, handle the response and parse the data into a format we can use.
+Using `XMLHttpReqest` directly is a handy way to access remote data, or web APIs, but it requires us to do a lot of lower-level wiring to get everything working. Now we're going to take a look at the Ajax capabilities of jQuery, which abstracts the XHR code into a nice, higher-level package.
```
Comment (pletcher) [inline README.md]: Maybe we can make this a bit more abstract by changing the first sentence to some thing like,

"In JavaScript, we can use `XMLHttpRequest` directly to access remote data, or web APIs, but..."

File: README.md [hunk 2]
```diff
@@ -1,168 +1,187 @@
...
-<!doctype html>
-<head>
-  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js"></script>
-    <script src="script.js"></script>
-</head>
 <body>
   <p id="sentences">
     Loading our page...
   </p>
+  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js"></script>
```
Comment (pletcher) [inline README.md]: Could we update this to jQuery 3.0.0?

**Comment by pletcher** [comment]
The de-railsification is A+ -- thank you!

File: README.md (hunk 2 above)
Comment (scottcreynolds) [inline README.md]: hahaha I'm dumb. I updated it in the code in the HTML page, and then of course deleted all that code so they could code along.

**Comment by pletcher** [comment]
Thanks!

**Pull request merged**